stored_queries/ncbi_tax/ncbi_taxon_get_siblings.yaml

```diff
@@ -0,0 +1,26 @@
+# Get the array of siblings for a taxon
+# Results are limited to 10k
+name: ncbi_taxon_get_siblings
+params:
+  type: object
+  required: [key]
+  properties:
+    key:
+      type: string
+      title: Document key
+      description: Key of the taxon vertex for which you want to find siblings
+query: |
+  // First get the id of the parent node
+  let parent_id = FIRST(
+    for t in ncbi_taxon
+      filter t._key == @key
+      for p in 1..1 OUTBOUND t ncbi_child_of_taxon
+        return p._id
+  )
+  // Get all children from the parent, filter out @key
+  for p in ncbi_taxon
+    filter p._id == parent_id
+    for c in 1..1 INBOUND p ncbi_child_of_taxon
+      filter c._key != @key
+      limit 10000
```

Comment:
I assume you're not willing to add a test for the limit

Reply:
I don't think we need to